### ISSUE TITLE
Why have a dependency on okio?

### ISSUE THREAD

**Issue by mirraj2** [rockaport/alice#11]
You literally only use it in one place and it is EASILY replaced with a ByteArrayOutputStream.

I rewrote your the last part of your encrypt() method to not use any dependencies:

```java
ByteArrayOutputStream output = new ByteArrayOutputStream();
    try {
      output.write(cipher.getIV());
      output.write(encryptedBytes);

      // compute the MAC if needed and append the MAC (IV || CIPHER || MAC)
      if (context.getMacAlgorithm() != AliceContext.MacAlgorithm.NONE) {
        output.write(getMac(context.getMacAlgorithm(), password).doFinal(encryptedBytes));
      }
    } catch (IOException e) {
      throw new RuntimeException(e);
    }

    return output.toByteArray();
```

**Comment by rockaport** [comment]
Yep, it's unnecessary. The idea at the time was to integrate okio sources and sinks, but eventually that got tossed aside. Submit a PR and I'll get it merged.

**Comment by rockaport** [comment]
Fixed in v0.9.0
